### PR TITLE
docs: sample for limitToLast() query 

### LINKED ISSUE #1108: Add Sample for Query.limitToLast()
Sample for the feature introduced by https://github.com/googleapis/nodejs-firestore/pull/954

### PR DESCRIPTION
**Pull request by laljikanjareeya**
Fixes #1108

### CODE REVIEW

File: samples/test/limit-to-last-query.test.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    const firestore = new Firestore();
+    const cities = ['San Francisco', 'Los Angeles', 'Tokyo', 'Beijing'];
+
+    before(async () => {
+        await Promise.all(cities.map(city => firestore.doc(`cities/${city}`).create({ name: city })));
+    });
+
+    after(async () => {
+        const cityCollectionRef = firestore.collection('cities');
+        const cityDocs = (await cityCollectionRef.select('id').get()).docs;
```
Comment (schmidt-sebastian) [inline samples/test/limit-to-last-query.test.js:33]: 'id' is called '__name__' in the protocol. You should use the sentinel FieldValue.documentId(), which hides this from our users.

Comment (laljikanjareeya) [reply samples/test/limit-to-last-query.test.js:33]: @schmidt-sebastian you mean to use `FieldPath.documentId()`:
```ts
const cityDocs = (await cityCollectionRef.select(FieldPath.documentId()).get()).docs;
```

Comment (schmidt-sebastian) [reply samples/test/limit-to-last-query.test.js:33]: Yes, that's the one.

Comment (laljikanjareeya) [reply samples/test/limit-to-last-query.test.js]: Done!!

File: samples/test/limit-to-last-query.test.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const { execSync } = require('child_process');
+const { assert } = require('chai');
+const { describe, it } = require('mocha');
+const exec = (cmd) => execSync(cmd, { encoding: 'utf8' });
+const { Firestore,FieldPath } = require('@google-cloud/firestore');
```
Comment (schmidt-sebastian) [inline samples/test/limit-to-last-query.test.js:21]: The formatting seems off now. Can you add a space after the comma?

**Pull request merged**